src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php

```diff
@@ -153,6 +154,19 @@ public function load(array $configs, ContainerBuilder $container): void
             }
         }
 
+        if (!isset($config['cache']) || true === $config['cache']) {
```

Comment:
I'm not sure I follow here: why is true a special value?
where do we keep supporting string values?

Reply:
Shouldn't we support the current behavior even if we deprecate it ?

Reply:
Nevermind, I misread and misunderstood your comment. I need to check the code to verify what we do with string values.

Reply:
@nicolas-grekas I slightly updated the code to inject no cache in the Warmer if the cache option is a string, so that we fall back to the old behavior.

If we have `cache: false`, all the services we created must be removed. If `cache` is a string value, we remove the extra cache setup and reconfigure the CacheWarmer.
If `cache` is not provided, or `true`, the new behavior is used: we do the caching in the `build_dir` with a runtime cache in the `cache_dir`.

Does that seem better to you ?

Reply:
And this code can be simplified in 8.0, once we remove the `cache` key from the configuration.